go.mod

```diff
@@ -3,19 +3,18 @@ module sironamedical/vicimonitor
 go 1.21.0
 
 require (
-	github.com/mitchellh/go-ps v1.0.0
-	github.com/prometheus/client_golang v1.17.0
+	github.com/prometheus/client_golang v1.20.2
```

Comment:
Isn't the latest 1.23.0? or previous 1.22.6 if the latest may not be stable-enough.

Reply:
where do you see version 1.23? 

https://github.com/prometheus/client_golang/releases

Reply:
Wait, I read this as the Golang version 🤦 

nvm, sorry for the noise!